### PR TITLE
Added quiz screen with local data

### PR DESCRIPTION
**Pull request by aleke88884**
We added  a Quiz page with local data but it works, also we have some bugs in app i will fix them. But now it works simply.

There a pages:
1) At last item we added a button for the first time after we will change it.

image

2) About colors and buttons it just for the first time. We will add other ways of passing quiz like fields, chekboxes and etc.

image

3) The test result is displayed in a dialog box.
Also have two buttons "Restart" and "Back". By cliking "Restart" button user will start a quiz once more. By cliking "Back" button page changes to the theories page (where list view with theories). 

image

### CODE REVIEW

File: lib/features/quiz/model/quiz.dart [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+class Answer {
+  final String answerText;
+  final bool isCorrect;
+  Answer({required this.answerText, required this.isCorrect});
+}
+
+List<Question> getQuestions() {
+  List<Question> list = [];
+  // Adding some data
+  list.add(Question(
```
Comment (rzhaxibayev) [inline lib/features/quiz/model/quiz.dart]: why we need add them, can't we just create like list = [Question, Question];

File: lib/features/quiz/screen/quiz_screen.dart [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+class _QuizScreenState extends State<QuizScreen> {
+  List<Question> questionList = getQuestions();
+  int currentQuestionIndex = 0;
+  int score = 0;
+  Answer? selectedAnswer;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color.fromARGB(255, 55, 50, 80),
+      body: Container(
```
Comment (rzhaxibayev) [inline lib/features/quiz/screen/quiz_screen.dart]: Padding
Don't use Container only for margin, padding purposes, it's much heavier than Padding

File: lib/features/quiz/screen/quiz_screen.dart [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+            _questionWidget(),
+            _answerList(),
+            _nextButton()
+          ],
+        ),
+      ),
+    );
+  }
+
+  _questionWidget() {
```
Comment (rzhaxibayev) [inline lib/features/quiz/screen/quiz_screen.dart]: bad idea to create widgets with a method, move it into separate class class _QuestionWidget, class _AnswerList etc

Comment (rzhaxibayev) [reply lib/features/quiz/screen/quiz_screen.dart]: and create class with const constructor, that is much effective

File: lib/features/quiz/screen/quiz_screen.dart [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+              currentQuestionIndex ++;
+            });
+          }
+        },
+        child: const Text('Next'),
+      ),
+    );
+  }
+  
+  _showScore() {
```
Comment (rzhaxibayev) [inline lib/features/quiz/screen/quiz_screen.dart]: class

File: lib/features/quiz/model/quiz.dart (hunk 1 above)
Comment (aleke88884) [reply lib/features/quiz/model/quiz.dart]: yes sure i will change it

File: lib/features/quiz/screen/quiz_screen.dart (hunk 3 above)
Comment (aleke88884) [reply lib/features/quiz/screen/quiz_screen.dart]: ok

File: lib/features/quiz/screen/quiz_screen.dart (hunk 4 above)
Comment (aleke88884) [reply lib/features/quiz/screen/quiz_screen.dart]: ok

**Pull request merged**